client/src/ingredients/examples.js

```diff
@@ -92,3 +92,7 @@ export function Examples({ examples }) {
     </>
   );
 }
+
+export function Example({ example }) {
```

Comment:
Just put `export ` in front on line 65. 
Also, to align with the style of naming, would you mind renaming `RenderExample` to `Example`. and do the necessary renaming on line 90.

Reply:
Clearly the result of a late night PR..apologies 😅